test/charms/test_benchmark/src/bcharm.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
```

Comment:
How about something easier to understand in the directory listing than `bcharm` "be charmed?"

Given that it's the only file under `src`, why not `charm.py`?

If it must to be different, maybe `benchmark_charm.py`?

Reply:
> Given that it's the only file under `src`, why not `charm.py`?

We annoyingly have ops/charm.py and also default to charms being src/charm.py. Most of the time this is fine, but in the tests ops is in the `PYTHONPATH` so ops/charm.py gets found with a plain `import charm`. I originally had a bunch of path mangling to get `src` in the right place so that it'd pick up src/charm.py instead but it ended up pretty messy (I don't recall the specifics now).

> If it must to be different, maybe `benchmark_charm.py`?

Sure.